values/cert-manager/cert-manager.gotmpl

```diff
@@ -30,8 +32,12 @@ cainjector:
     limits:
       cpu: 200m
       memory: 384Mi
+  podAnnotations:
+{{- tpl $addIgnorePolicyAnnotation (list "psp-allowed-users" "psp-privileged" "psp-host-security") | nindent 4 }}
```

Comment:
this block can be put on the same line as `podAnnotations: `

Reply:
And please fix this in all `values/*.gotmpl` files.

Reply:
ok, done